### DISCUSSION TITLE
About towhee pipeline: Can I disable the env check when running my pipeline?

### DISCUSSION THREAD

**Discussion by LiuersNick** [towhee-io/towhee#2515] [Q&A] [answered]
Every time before i running the pipeline, it checks if the environment meets the requirements like:

![image](https://user-images.githubusercontent.com/59303811/236127602-31fc2569-e544-4486-9d74-7a35458efee0.png)

And I think this process takes some time. 
**I hope to disable this feature to to shorten the running time of my pipeline.** Are there any solutions?

Here is my code:
```python
def load_image_embedding(table_name):
    pipe_load = (
        pipe.input('table_name')
        .flat_map('table_name', ('id', 'path'), read_mysql)
        .map('path', 'img', ops.image_decode.cv2('rgb'))
        .map('img', 'vec', ops.image_text_embedding.clip(model_name='clip_vit_base_patch16', modality='image', device=0))
        .map('vec', 'vec', lambda x: x / np.linalg.norm(x))
        .map(('id', 'vec'), (), ops.ann_insert.milvus_client(host='127.0.0.1', port='19530', collection_name='text_image_search'))
        .output(tracer=True)
    )
    pipe_load(table_name)
    profiler_info = pipe_load.profiler()
    return profiler_info
```

Thanks for your reply!

**Comment by junjiejiangjjj** [comment, marked as answer]
The clip operator depends on `datasets[vision]`, currently, this format of dependency detection has problems, leading to an attempt to install every time.

**Reply by LiuersNick** [reply]
Okay, and thanks for you reply
